src/BloomBrowserUI/bookLayout/basePage.less

```diff
@@ -372,10 +372,14 @@ Changes here generally require similar changes in EpubMaker.FixPictureSizes().
     .Device16x9Landscape & {
         .SetMarginBoxDevice(@Device16x9Landscape-Width, @Device16x9Landscape-Height);
 
-        //make the top half take up the whole screen
-        .position-top{bottom: 0 !important;}
-        .position-bottom{display:none;}
+        //that makes the margin box actually have no margin. Which looks good for the image, but
+        //looks bad for the text. So here we re-introduce a margin... a bit of a hack
+        .bloom-editable{
+            margin-left: 10px;
```

Comment:
Go ahead and introduce a less variable, as we do in code. E.g. @marginForDevices